### PR TITLE
LINQPad5 to v5.44.02

### PR DESCRIPTION
**Pull request by MarcusP-P**
Upgrade to the latest LINQPad5. Because the installer gets replaced, the current manifest no longer works.

- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/23264)

### CODE REVIEW

File: manifests/l/LINQPad/LINQPad/5/5.44.02/LINQPad.LINQPad.5.locale.en-US.yaml [hunk 1]
```diff
@@ -1,13 +1,12 @@
-﻿# Created using YamlCreate.ps1
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
 
 PackageIdentifier: LINQPad.LINQPad.5
-PackageVersion: 5.43.00
+PackageVersion: 5.44.02
```
Comment (ItzLevvie) [inline manifests/l/LINQPad/LINQPad/5/5.44.02/LINQPad.LINQPad.5.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 5.44.2
```

File: manifests/l/LINQPad/LINQPad/5/5.44.02/LINQPad.LINQPad.5.yaml [hunk 2]
```diff
@@ -1,8 +1,9 @@
-﻿# Created using YamlCreate.ps1
+# Created using wingetcreate 0.3.0.3
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
 
 PackageIdentifier: LINQPad.LINQPad.5
-PackageVersion: 5.43.00
+PackageVersion: 5.44.02
```
Comment (ItzLevvie) [inline manifests/l/LINQPad/LINQPad/5/5.44.02/LINQPad.LINQPad.5.yaml:5]:
```suggestion
PackageVersion: 5.44.2
```

File: manifests/l/LINQPad/LINQPad/5/5.44.02/LINQPad.LINQPad.5.installer.yaml [hunk 3]
```diff
@@ -1,24 +1,23 @@
-﻿# Created using YamlCreate.ps1
+# Created using wingetcreate 0.3.0.3
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
 
 PackageIdentifier: LINQPad.LINQPad.5
-PackageVersion: 5.43.00
+PackageVersion: 5.44.02
```
Comment (ItzLevvie) [inline manifests/l/LINQPad/LINQPad/5/5.44.02/LINQPad.LINQPad.5.installer.yaml:5]:
```suggestion
PackageVersion: 5.44.2
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
**You'll have to update the version folder as well.**

**Comment by MarcusP-P** [comment]
> You'll have to update the version folder as well.

Oh thanks. Fixed

**Review by ItzLevvie — APPROVED** [review]
image

**Pull request merged**